frontend/@types/allowance.ts

```diff
@@ -0,0 +1,28 @@
+import { Asset, SubAccount } from "@redux/models/AccountModels";
+import { Contact } from "@redux/models/ContactsModels";
+
+export interface Allowance {
```

Comment:
Why is  not used the zod interface creator, the same with the enums

Reply:
Done